doc/parameters/param_list.md

```diff
@@ -674,3 +674,18 @@ When `recovery_plugins` parameter is not overridden, the following default plugi
 | min_speed | 0.0 | Minimum speed (m/s) |
 | max_speed | 0.5 | Maximum speed (m/s) |
 | filter_duration | 0.3 | Duration (secs) for velocity smoothing filter |
+
+### BT Node TruncatePath
+
+| Input Port | Default | Description |
+| ---------- | ------- | ----------- |
+| input_path | N/A | Path to be truncated |
```

Comment:
Update the names / values here in the navigation.ros.org PR

Reply:
Ok. I am planning to do it in that PR after this PR.